bazel/scripts/check_py_bazel.sh

```diff
@@ -46,6 +46,7 @@ DENY_LIST_NOT_RELEVANT=(
   "./show-tech"
   "./third_party"
   "./xwf/gateway/deploy"
+  "./hil_testing"
```

Comment:
Could we keep this list sorted alphabetically, i.e. `"./hil_testing"` after `"./feg/gateway/docker"`?